test/lazy/test_reuse_ir.py

```diff
@@ -101,24 +101,5 @@ def testAddSubFallback(self):
         torch._lazy.ir_cache.reset()
         torch._lazy.config.set_force_fallback("")
 
-    def testBatchNorm(self):
-        device = get_test_device()
-        x = torch.randn(16, 3, 224, 224, device=device)
-        bn = torch.nn.BatchNorm2d(3).to(device=device)
-        for i in range(10):
-            z = bn(x)
-
-        device = "lazy"
-        x_lazy = x.detach().clone().to(device=device)
-        bn = bn.to(device=device)
-        for i in range(10):
-            z_lazy = bn(x_lazy)
-            torch._lazy.mark_step()
-
-        torch.testing.assert_close(z.cpu(), z_lazy.cpu())
-        assert metrics.counter_value("IrNodeReused_torch::lazy::TSNativeBatchNormForward") >= 7
```

Comment:
Can we maybe not remove this test altogether but rather replace the instance of `TsNativeBatchNormForward` with the codegen requivalent?

Reply:
That's a good point, I'll try that out